models/edr/dbt_artifacts/dbt_exposures.sql

```diff
@@ -1,6 +1,6 @@
 {{
   config(
-    materialized='table',
+    materialized='incremental',
```

Comment:
Why?

Reply:
I've made the artifact models incremental so that their contents won't necessarily be deleted when the model is run and by that force the re-upload of the same data. A common case is that the artifacts don't change at all.

Reply:
I don't think it's a problem, most of our users don't run it that often unless we explicitly tell them to do so. And in that case it's ok that it will take the additional upload time IMO

Reply:
This means that if a user has `dbt run -s elementary` as part of his pipeline, it'll necessarily re-upload all the artifacts and will never utilize the cache. Are we okay with that?